README.md

```diff
@@ -22,6 +22,12 @@ An implementation of VHD [VHD specification](https://technet.microsoft.com/en-us
     2. https://github.com/Azure/azure-sdk-for-go
             This package is used to make Microsoft Azure storage API calls.
 
+# Features
+
+    1. Fast uploads - This tool offers faster uploads than the [Azure Xplat CLI](https://github.com/Azure/azure-xplat-cli).
```

Comment:
meh, unindenting is better for reading on mobile etc. code blocks will get a scrollbar.